libkbfs/block_journal.go

```diff
@@ -1018,32 +647,27 @@ func (j *blockJournal) removeFlushedEntry(ctx context.Context,
 
 	// Remove any of the entry's refs that hasn't been modified by
 	// a subsequent block op (i.e., that has earliestOrdinal as a
-	// tag).
+	// tag). Doesn't have an effect for removeRefsOp, since those
+	// are already removed, and mdRevMarkerOp, but doing this is
```

Comment:
The wording on this sentence is a bit confusing.  Maybe something like:

> Has no effect for removeRefsOp (since those are already removed) or mdRevMarkerOp (which has no references).